src/components/Book.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ProtoType from "prop-types";
+
+const Books = props => {
+    // eslint-disable-next-line react/prop-types
```

Comment:
Kindly avoid disabling the ESLint inline rules, it is almost the same as altering the eslintrc.json file or any of the linter files :+1: